src/pat/inject/inject.js

```diff
@@ -37,7 +37,7 @@ define([
 
     var _ = {
         name: "inject",
-        trigger: "a.pat-inject, form.pat-inject, .pat-subform.pat-inject",
+        trigger: ".ui-button.pat-inject, a.pat-inject, form.pat-inject, .pat-subform.pat-inject",
         init: function inject_init($el, opts) {
```

Comment:
Ok, so this is Raptor specific.

Is the `.pat-raptor` element perhaps a parent of the `div.ui-button` element? Then the selector could be made `.pat-raptor .ui-button.pat-inject`.

That's nicely descriptive. It's clear that this trigger has to do with pat-raptor and buttons clicked.

If `.pat-raptor` is not available, perhaps another class which indicates that this is relevant to Raptor.